app/serializers/cache/search_cache_methods.rb

```diff
@@ -1,21 +1,34 @@
 module Cache
   module SearchCacheMethods
-    def has_valid_dates(hash, start_key = :validity_start_date, end_key = :validity_end_date)
```

Comment:
This start_key/end_key behaviour was a copy/paste from a previous implementation that was used to serialize measures